zipline/pipeline/visualize.py

```diff
@@ -174,8 +174,8 @@ def writeln(f, s):
 
 def fmt(obj):
     if isinstance(obj, Term):
-        if hasattr(obj, 'short_repr'):
-            r = obj.short_repr()
+        if hasattr(obj, 'graph_repr'):
+            r = obj.graph_repr()
```

Comment:
don't all terms have this because it is defined on `Term`

Reply:
Other stuff runs through this function as well (e.g., non-term parameters).

Reply:
`if isinstance(obj, Term):`

Reply:
Ah. idk then. This change was just made with find-replace. This is probably just a holdover from before I put a default impl on short_repr.